### PR TITLE
Chore/upgrade ember

### PR DESCRIPTION
**Pull request by Alonski**
Merge in after: #38

### CODE REVIEW

File: app/templates/application.hbs.orig [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<<<<<<< HEAD
```
Comment (patocallaghan) [inline app/templates/application.hbs.orig]: 🔪 this

File: config/optional-features.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "jquery-integration": true
```
Comment (patocallaghan) [inline config/optional-features.json]: Do we need jQuery?

File: app/templates/application.hbs.orig (hunk 1 above)
Comment (Alonski) [reply app/templates/application.hbs.orig]: Doh I missed it

File: config/optional-features.json (hunk 2 above)
Comment (Alonski) [reply config/optional-features.json]: Probably not. I'll fix tomorrow

**Pull request merged**